test/verify/check-packagekit

```diff
@@ -67,7 +67,7 @@ class TestUpdates(MachineCase):
 
         # create two updates
         self.createPackage("vanilla", "1.0", "1", install=True)
-        self.createPackage("vanilla", "1.0", "2")
+        self.createPackage("vanilla", "1.0", "2", postinst='sleep 10')
```

Comment:
I'm glad you introduced the finish-pk lockfile ealier; moving to this uncovered a race condition in the test, as we can't assume that chocolate gets installed before vanilla. But the test relies on this, so I added dependencies to the test packages. Now both the ordering and the sleep races are fixed.

Force-pushed the update.